python/cudf/cudf/core/frame.py

```diff
@@ -3295,6 +3301,211 @@ def _reindex(
         return self._mimic_inplace(result, inplace=inplace)
 
 
+class FrameOneD(Frame):
```

Comment:
Rather than `FrameOneD` which is somewhat ambiguous (one row? one column? either?), how about something like `SingleColumnFrame`?